lib/appboy/rest/schedule_messages.rb

```diff
@@ -15,6 +15,31 @@ def initialize(app_group_id, send_at:, messages: [], segment_id: nil, external_u
       end
 
       def perform
+        if schedule_id.nil?
+          create_scheduled_push
+        else
+          update_scheduled_push
+        end
+      end
+
+      private
+
+      def update_scheduled_push
+        http.post('/messages/schedule/update', {
+          app_group_id: app_group_id,
+          schedule_id: schedule_id,
+          external_user_ids: external_user_ids,
+          segment_id: segment_id,
+          campaign_id: campaign_id,
+          messages: messages,
+          schedule: {
+            time: send_at,
+            in_local_time: local_timezone
+          }
```

Comment:
Let's break these parameters into their own method and call that method for both create and update methods.